cereslib/events/events.py

```diff
@@ -119,25 +119,25 @@ def _add_sh_info(self, df_columns, item, update_sh_db=False):
         author_bot = False
 
         # Add SH information, if any
-        if sh_identity[Events.SH_AUTHOR_ID]:
+        if Events.SH_AUTHOR_ID in sh_identity and sh_identity[Events.SH_AUTHOR_ID]:
```

Comment:
What I mention in my review is merging this line and lin 113 with something like:
```
author_id = sh_identity.get(Events.SH_AUTHOR_ID, Events.UNKNOWN)
```
And similarly for the next ones, just taking care of `author_bot` that should be `False` by default.

Reply:
sure!